### ISSUE TITLE
Should we document in the spec that there is no consensus on whether supporting MediaStreamTrackProcessor for audio?

### ISSUE THREAD

**Issue by youennf** [w3c/mediacapture-transform#92]
Should we document in the spec that there is no consensus on whether supporting MediaStreamTrackProcessor for audio?

**Comment by youennf** [comment]
The spec says:

```
Note: There is no WG consensus on whether or not audio use cases should be supported.
```

It is precise about audio track generator.
But it is not precise about MediaStreamTrackProcessor.
It might be good to add a note similar to the one done for VideoTrackGenerator.

**Comment by youennf** [comment]
I uploaded https://github.com/w3c/mediacapture-transform/pull/104 which partially fixes the issue.
What remains is what to do when MediaStreamTrackProcessor constructor is given an audio track.
The spec should probably say something, safari current implementation throws.

@jan-ivar, @alvestrand, thoughts?

**Comment by youennf** [comment]
I guess audio track throwing may be already allowed since it might not be considered as an invalid track.
This could be tackled in https://github.com/w3c/mediacapture-transform/issues/94.

**Comment by youennf** [comment]
> We should reopen this debate

That is a different issue, that we can tackle on with WG discussions.

The issue here is specifically about representing in the spec the current state of discussions, nothing less, nothing more. This is more editorial than anything else (except the part to represent lack of audio support).

**Comment by guidou** [comment]
> That is a different issue, that we can tackle on with WG discussions.
> 
> The issue here is specifically about representing in the spec the current state of discussions, nothing less, nothing more. This is more editorial than anything else (except the part to represent lack of audio support).

Absolutely. I agree that the spec should reflect that and your PR LGTM.
